tests/ccb-tests.js

```diff
@@ -8,20 +8,21 @@ function isNumeric(n) {
   return !isNaN(parseFloat(n)) && isFinite(n);
 }
 
-describe.skip('when executing ccb', function() {
+describe('when executing ccb', function() {
+  this.timeout(10000);
+
   var result;
+  // README: Add your jira username and password to this object before running tests.
```

Comment:
Can you read them from `process.env`? Then running the tests would be something like:

    $ JIRA_USERNAME=$JIRA_USERNAME JIRA_PASSWORD=$JIRA_PASSWORD npm test